loky/backend/_win_reduction.py

```diff
@@ -93,4 +93,5 @@ def _reduce_socket(s):
 def _rebuild_socket(ds):
     return ds.detach()
```

Comment:
Good call, it was not working properly at all. I fixed it in the last commit:

  * It is working for all version > 3.3, both at spawning time and in Queue for socket/connection
  * No socket/connection pickling possible between started process in version <= 3.3 but it is working properly when we are spawning a new process.
  * For windows, I removed the pickling of `Connection` object as it is never used in the `multiprocessing` API. I can add it back but I don't know how it should be used.